code/game/gamemodes/cult/ritual.dm

```diff
@@ -239,24 +251,53 @@ This file contains the arcane tome files as well as innate cultist emergency com
 			break
 	if(!rune_to_scribe)
 		return
+	var/turf/Thenewturfyouwalkedto = get_turf(user) //we may have moved. adjust as needed...
+	if(locate(/obj/effect/rune) in Thenewturfyouwalkedto)
+		user << "<span class='cult'>There is already a rune here.</span>"
+		return
 	if(!Adjacent(user) || !src || qdeleted(src) || user.incapacitated())
 		return
 	user.visible_message("<span class='warning'>[user] cuts open their arm and begins writing in their own blood!</span>", \
 						 "<span class='cult'>You slice open your arm and begin drawing a sigil of the Geometer.</span>")
 	if(iscarbon(user))
 		var/mob/living/carbon/C = user
 		C.apply_damage(0.1, BRUTE, pick("l_arm", "r_arm"))
-		if("Call Forth The Geometer" == entered_rune_name)
+		if("Summon Nar-Sie" == entered_rune_name)
+			var/confirm_final = alert(usr, "This is the FINAL step to summon Nar-Sie, it is a long, painful ritual and the crew will be alerted to your presence", "Are you prepared for the final battle?", "My life for Nar-Sie!", "No")
+			if(confirm_final == "No")
+				usr << "On second thought, we should prepare further for the final battle..."
+				return
 			C.apply_damage(40, BRUTE, pick("l_arm", "r_arm"))
 			var/area/A = get_area(src)
 			var/locname = initial(A.name)
-			priority_announce("Figments from an eldritch god are being summoned by [user] into [locname] from an unknown dimension. Disrupt the ritual before it reaches a critical point.","Central Command Higher Dimensionsal Affairs")
-			if(!do_after(user, 500, target = get_turf(user)))
+			priority_announce("Figments from an eldritch god are being summoned by [user] into [locname] from an unknown dimension. Disrupt the ritual at all costs!","Central Command Higher Dimensionsal Affairs", 'sound/AI/spanomalies.ogg')
+			for(var/turf/B in orange (1, user))
+				var/obj/machinery/shield/N = new(B)
+				N.name = "Rune-Scriber's Shield"
+				N.desc = "A potent shield summoned by cultists to protect them while they prepare the final ritual"
+				N.color = "red"
```

Comment:
Instead of this, set the icon_state to "shield-red", which will produce more red-looking shields.
Alternatively, set it to "shield-grey" and the color to whatever hex color you want for it.